src/plays/motivate-me/components/cards/index.css

```
@@ -0,0 +1,41 @@
+.card-container-main{
```

@Hat52, Add prefix to your CSS classes so that it won't conflict with other play CSS classes.

Please follow the same structure everywhere.

```suggestion
.motivation_quote_card-container-main{
```

---

@Hat52 , Did you change this as well?

---

yes, I have changed this as well.